hibernate-examples/src/test/resources/hibernate.properties

```diff
@@ -21,15 +21,15 @@
 # 51 Franklin Street, Fifth Floor
 # Boston, MA  02110-1301  USA
 #
-hibernate.dialect org.hibernate.dialect.H2Dialect
-hibernate.connection.driver_class org.h2.Driver
-hibernate.connection.url jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1;MVCC=TRUE
-hibernate.connection.username sa
-hibernate.connection.password
+hibernate.dialect=org.hibernate.dialect.H2Dialect
+hibernate.connection.driver_class=org.h2.Driver
+hibernate.connection.url=jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1;MVCC=TRUE
+#hibernate.connection.username=sa
```

Comment:
by *not* setting the connection username and password Hibernate no longer tries to use the multi-pool get connection method.